tox.ini

```diff
@@ -26,3 +26,10 @@ commands =
     !integration: pytest --junitxml=.tox/junit-{envname}.xml test/unit
 # Run integration test if in intgration env
     integration: ansible-playbook {posargs}
+
+[flake8]
+# These are things that the devs don't agree make the code more readable
+# E402 module level import not at top of file
+ignore = E402
+# not all the devs believe in 80 column line length
+max-line-length = 160
```

Comment:
Currently, the `flake8` command takes a lot of time because it also analyzes `venv` folder.

Please, add `exclude` argument to ignore non-needed files and dirs. At least, we should ignore `.git`, `.venv`, `venv`, `__pycache__`, but feel free to extend this list.

Reply:
fixed